<launch>
  <node pkg="rosduct" name="rosduct" type="rosduct_main.py" output="screen" respawn="true">
    <rosparam>
    # ROSbridge websocket server info
    rosbridge_ip: localhost
    rosbridge_port: 9090
    # Topics being published remotely to expose locally
    remote_topics: [
                   ]
    # Topics being published in the local roscore to expose remotely
    local_topics: [
                        ['/map', 'nav_msgs/OccupancyGrid']
                  ]
    # Services running remotely to expose locally
    remote_services: [
                        ['/task_server/register_agent', 'long_term_deployment/RegisterAgent'],
                        ['/task_server/unregister_agent', 'long_term_deployment/UnregisterAgent'],
                        ['/task_server/get_agents', 'long_term_deployment/GetRegisteredAgents']
                     ]
    # Services running locally to expose remotely
    local_services: [
                    ]
    # Parameters to be sync, they will be polled to stay in sync
    #parameters: []
    #parameter_polling_hz: 1

    </rosparam>
  </node>
  <node pkg="long_term_deployment" name="robot_client" type="client.py" output="screen">
    <param name="agent_name" value="turtlebot"/>
  </node>
</launch>
